columnq/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(warnings)]
+#![allow(clippy::result_large_err)]
```

Comment:
i have pushed a fix on the main brain to fix this error so we don't need to specify the allow here, could you rebase to pick up the fix?